clusterloader2/pkg/util/util.go

```diff
@@ -71,7 +71,12 @@ func GetMap(dict map[string]interface{}, key string) (map[string]interface{}, er
 	return getMap(dict, key)
 }
 
-// GetStringArray tries to return value from map of type map with casting to a string using fmt.Sprintf. If value doesn't exist, error is returned.
+// GetMapArray tries to return value from map of type []map. If value doesn't exist, error is returned.
+func GetMapArray(dict map[string]interface{}, key string) ([]map[string]interface{}, error) {
+	return getMapArray(dict, key)
+}
+
+// GetStringArray tries to return value from map cast to a string using fmt.Sprintf. If value doesn't exist, error is returned.
```

Comment:
Are you sure you want to change this comment?

Reply:
I think I did, but not to this :) I meant cast to a string slice (as it doesn't return a map, nor accept a map of maps). Does that make sense?

Reply:
Right, it should include a string slice information. Thanks.